src/schemas.py

```diff
@@ -10,3 +10,16 @@ class Project(BaseModel):
 
 class ProjectDetails(Project):
     project_id: uuid.UUID = Field(default_factory=uuid.uuid4)
+
+
+class User(BaseModel):
+    user_id: uuid.UUID = Field(default_factory=uuid.uuid4)
+    name: str
+    email: EmailStr
+    password: str
```

Comment:
worth adding basic min len requirement, later you could improve that on auth step further